docs/source/installation.rst

```diff
@@ -1,6 +1,8 @@
 Installation Guide
 ==================
 
+**Author**: `Priya Goyal <https://github.com/prigoyal>`_
```

Comment:
I added this for support purposes, when people read docs, they can contact on github/slack immediately.